src/Resources/config/block.xml

```diff
@@ -11,35 +11,35 @@
     <services>
         <service id="sonata.block.service.container" class="%sonata.block.service.container.class%">
             <tag name="sonata.block"/>
-            <argument/>
-            <argument type="service" id="templating"/>
+            <argument>sonata.block.container</argument>
```

Comment:
carefull with all this. It was removed on master on purpose: https://github.com/sonata-project/SonataBlockBundle/pull/302

cc/ @core23